app/views/templates/decision_template.scala.html

```diff
@@ -78,7 +78,7 @@
+            <img class="crown-logo-atar" alt="HM Revenue and Customs" src="@messages("logo.encoded")"/>
```

Comment:
Please remove `class="crown-logo-atar"`. There is no styling associated with that class in this file but there is in the other two templates.